geom/geom/inc/TGeoVolume.h

```diff
@@ -368,4 +365,16 @@ inline Int_t TGeoVolume::GetNdaughters() const
    return (fNodes->GetEntriesFast());
 }
 
+inline Char_t TGeoVolume::GetTransparency() const
+{
+   // If the transparency is (-1), the old default handling is applied
+   if ( fTransparency >= 0 ) return fTransparency;
+   return !fMedium ? 0 : fMedium->GetMaterial()->GetTransparency();
+}
+
+inline void TGeoVolume::SetTransparency(Char_t transparency)
+{
+   fTransparency = transparency;
+}
```

Comment:
This is unfortunately not backward compatible with the previous `SetTransparency` behavior, because it will only modify the transparency of the respective volume so that all previous TGeo visualization scripts will produce different pictures. We have to leave SetTransparency untouched and add a method with a different name:
```
/// @brief Reset transparency for this volume only. Calling with no argument restores material-based transparency
void ResetTransparency(char transparency = -1) { fTransparency = transparency) // *MENU*
```

Reply:
OK. Requested change applied.

Markus